libs/core/src/lib/menu/menu.component.ts

```diff
@@ -69,6 +69,13 @@ export class MenuComponent
     @Input()
     focusTrapped = true;
 
+    /**
+     * Whether the popover should automatically move focus into the trapped region upon
+     * initialization and return focus to the previous activeElement upon destruction.
+     */
+    @Input()
+    focusAutoCapture = true;
```

Comment:
Why we need this? 

We should not be able to change the correct behavior. Dropdown opened - focus moved to its first item, it's a rule of thumb on my opinion.

Reply:
Yes, ultimately, it's same behavior, but in addition, it enables users to prevent focus on first item by making `focusAutoCapture` false. It's same as in `Action Sheet` component. 
@platon-rov

Reply:
Okay, but where it is used?